stable/graylog/templates/tcp-service.yaml

```diff
@@ -21,12 +21,10 @@ spec:
       port: {{ .port }}
       protocol: TCP
       targetPort: {{ .port }}
+    {{- if .nodePort }}
```

Comment:
The `service.type` for `NodePort` is still need to check. You can add nodeport to other service type.

For signing off
https://github.com/helm/charts/pull/14008/checks?check_run_id=134192087